listener.go

```diff
@@ -0,0 +1,27 @@
+package mesg
+
+import "context"
+
+// Listener is a task execution listener.
+type Listener struct {
+	// Err filled when Listener fails to continue.
+	Err chan error
```

Comment:
I think we should definitely keep this and add to docs. mesg-js also has this with `stream.on('error', handler)` syntax. It is good to know when a listener/stream ends with an error.

Reply:
I'm 100% agree with it's better to return error to user.
But, I think we should take time and think about the error management in a global way. It could be a 1.1 released in a few days.
For instance, "Err" is not a proper name. So if we expose it now, and rename it in a few days, it will break the API. Better to not expose it now so that if we rename it, it doesn't break public API.